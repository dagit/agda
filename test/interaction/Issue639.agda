module Issue639 where

postulate A : Set
