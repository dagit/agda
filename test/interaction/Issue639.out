Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue639 (Issue639.agda).\n" t)
(agda2-info-action "*Error*" "Issue639.agda:3,11-18\nCannot postulate A with safe flag\nwhen scope checking the declaration\n  A : Set" nil)
((last . 3) . (agda2-goto '("Issue639.agda" . 34)))
(agda2-status-action "")
Agda2> 